updatecli/updatecli.d/alpine-linux.yml

```diff
@@ -25,7 +25,7 @@ sources:
       versionFilter:
         kind: regex
```

Comment:
```suggestion
        kind: semver
```

Reply:
- As per: https://www.updatecli.io/docs/plugins/github_release/#_semver , it avoids specifying a regex pattern and keep the expected version format for Alpine while always trying to update to latest